src/nidigital/metadata/functions.py

```diff
@@ -94,7 +94,7 @@
                 'direction': 'in',
                 'name': 'offsets',
                 'size': {
-                    'mechanism': 'passed-in',
+                    'mechanism': 'len',
```

Comment:
Please confirm that this edit came from internal NI metadata, was not done on GitHub alone.

Reply:
Yes, there's a review request (317029) for internal metadata change.